package.json

```diff
@@ -62,6 +62,9 @@
   "license": "MIT",
   "main": "lib/index.js",
   "oclif": {
+    "topics": {
+      "plugins": { "description": "add/remove/create CLI plugins"}
```

Comment:
Can we also add a `longDescription` (maybe `Add, remove, or create CLI plug-ins.`)? Also, our style guidelines say to hyphenate `plug-ins`, so the `description` should be `add/remove/create CLI plug-ins`.

Reply:
👍 @ruthemmanuelle , I will add the hyphen to `plug-ins`. One question about the `longDescription`. Where does that get displayed?

Reply:
The `longDescription` displays in the reference doc (https://developer.salesforce.com/docs/atlas.en-us.sfdx_cli_reference.meta/sfdx_cli_reference/cli_reference.htm). Currently the `plugins` commands don't show up there, but Thomas has a story to include commands other than the standard set (e.g. other teams' plug-ins), so we might as well add the `longDescription` now in case we can use it in the future.